.github/workflows/mac-run-tests-build-docs.yml

```diff
@@ -55,7 +55,8 @@ jobs:
 
     - name: Install Everest and dependencies
       run: |
-        pip install .[test, docs]
```

Comment:
Could it be that just removing the space works, or did you try that?

Reply:
I did not try that :disappointed: I am sure that would have worked